crates/breez-sdk/core/src/sync/mod.rs

```diff
@@ -0,0 +1,27 @@
+#![allow(dead_code)]
+use crate::SdkError;
+
+mod spark;
+mod sparkscan;
+
+pub(crate) use spark::SparkSyncService;
+pub(crate) use sparkscan::SparkscanSyncService;
+
+#[macros::async_trait]
+pub trait SyncService: Send + Sync {
+    async fn sync_payments(&self) -> Result<(), SdkError>;
+}
+
+pub enum SyncStrategy {
```

Comment:
What is the advantabe of the SyncStrategy enum? Vs just using the SyncService trait?

Reply:
I believe this makes it static instead of dynamic dispatch